documentation/modules/auxiliary/dos/http/slow_loris.md

```diff
@@ -0,0 +1,43 @@
+## Vulnerable Application
+
+This module tries to keep many connections to the target web server open and hold them open as long as possible.
+
+Vulnerable app versions include:
+
+- Apache HTTP Server 1.x and 2.x
+- Apache Tomcat 5.5.0 through 5.5.29, 6.0.0 through 6.0.27 and 7.0.0 beta
+
+Download the Metasploitable 2 vulnerable Linux virtual machine from [https://sourceforge.net/projects/metasploitable/files/Metasploitable2/](https://sourceforge.net/projects/metasploitable/files/Metasploitable2/).
```

Comment:
Is this to indicate how a user might verify the module? Perhaps move to step one and enhance the documentation in this area to make it more clear that the user could demonstrate one of the vulnerable applications by downloading and setting up the Metasploitable 2 virtual machine.